test/support/utils.rb

```diff
@@ -71,7 +69,8 @@ def debug_code(program, boot_options: '-r debug/start', remote: true, &block)
       else
         debug_on_local boot_options, TestInfo.new(dup_scenario)
       end
-      assert true
```

Comment:
I think this change will confuse users. If they see the following results, they don't know if test works well.

```shell
$ ruby test/debug/quit_test.rb
Tests on local and remote. You can disable remote tests with RUBY_DEBUG_TEST_NO_REMOTE=1.
Loaded suite test/debug/quit_test
Started
...
Finished in 1.047331 seconds.
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
3 tests, 0 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
2.86 tests/s, 0.00 assertions/s
```

Reply:
Ah I thought this is just a redundant line. But why aren't the previous assertion calls counted?

Reply:
https://github.com/ruby/debug/pull/180#issue-690609876 should help you.

Reply:
I see it makes sense now. Great work reducing the testing time btw 👍

Reply:
I've removed this change.